blog/what-is-source-code-verification/index.mdx

```diff
@@ -0,0 +1,733 @@
+
+At the deploy time, these 20 bytes will be replaced with the contract's own address.
+
+See [Solidity docs](https://docs.soliditylang.org/en/v0.8.28/contracts.html#call-protection-for-libraries).
+
+<details>
+<summary> Example </summary>
+
+In our previous library example we can look at the library `Lib`'s (deployed at [0x99F7C0086ab897C8fE65eBfF268c732a7e15b25F](https://holesky.otterscan.io/address/0x99F7C0086ab897C8fE65eBfF268c732a7e15b25F)) bytecodes. Since the call protection is not present before the deployment, i.e. in the creation bytecode, we'll only look at the runtime bytecodes:
+
+Recompiled runtime bytecode:
+```
+0x7300000000000000000000000000000000000000003014608060405260043610610034575f3560e01c8063cad0899b14610038575b5f80fd5b610052600480360381019061004d91906100b4565b610068565b60405161005f9190610101565b60405180910390f35b5f81836100759190610147565b905092915050565b5f80fd5b5f819050919050565b61009381610081565b811461009d575f80fd5b50565b5f813590506100ae8161008a565b92915050565b5f80604083850312156100ca576100c961007d565b5b5f6100d7858286016100a0565b92505060206100e8858286016100a0565b9150509250929050565b6100fb81610081565b82525050565b5f6020820190506101145f8301846100f2565b92915050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52601160045260245ffd5b5f61015182610081565b915061015c83610081565b92508282019050808211156101745761017361011a565b5b9291505056fea26469706673582212209d56669aec61ac1867a59a6d0472bd74975630174bc6e74d7e6881bba174c30564736f6c634300081a0033
+```
+
+Onchain runtime bytecode:
+```
+0x7399f7c0086ab897c8fe65ebff268c732a7e15b25f3014608060405260043610610034575f3560e01c8063cad0899b14610038575b5f80fd5b610052600480360381019061004d91906100b4565b610068565b60405161005f9190610101565b60405180910390f35b5f81836100759190610147565b905092915050565b5f80fd5b5f819050919050565b61009381610081565b811461009d575f80fd5b50565b5f813590506100ae8161008a565b92915050565b5f80604083850312156100ca576100c961007d565b5b5f6100d7858286016100a0565b92505060206100e8858286016100a0565b9150509250929050565b6100fb81610081565b82525050565b5f6020820190506101145f8301846100f2565b92915050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52601160045260245ffd5b5f61015182610081565b915061015c83610081565b92508282019050808211156101745761017361011a565b5b9291505056fea26469706673582212209d56669aec61ac1867a59a6d0472bd74975630174bc6e74d7e6881bba174c30564736f6c634300081a0033
+```
+
+</details>
+
+## After matching
+
+If we got a match, it means we can start from recompiled bytecodes, apply each runtime or creation transformations on top of the respective recompiled bytecodes, and obtain the onchain bytecodes. It also means that the provided source code and metadata can be associated with the contract at this chainId and address and we mark this contract as verified.
```

Comment:
I think here we should first say - "It means that the provided source code and metadata can be associated with the contract at this chainId and address and we mark this contract as verified.
Comment" and then "it also means we can start from recompiled bytecodes, apply each runtime or creation transformations on top of the respective recompiled bytecodes, and obtain the onchain bytecodes."